db/seeds.rb

```diff
@@ -1,4 +1,4 @@
-john = User.create(username: 'user',password: 'password', first_name: 'John', last_name: 'Doe', display_name: "MikeDaoSellsCelluliteProducts", email: 'example@example.com', role: 1)
+john = User.create(username: 'user',password: 'password', first_name: 'John', last_name: 'Doe', email: 'example@example.com', role: 1)
```

Comment:
Useless assignment to variable - `john`.<br>Line is too long. [134/80]<br>Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.